students/grzegorz_bajorski/lesson_01_python_basic/car_route.py

```diff
@@ -0,0 +1,10 @@
+import math
```

Comment:
There is no need to import the whole module. If you want to use just one particular function from the module:
from math import ceil